legend-engine-protocol-relational/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/store/relational/connection/DatabaseType.java

```diff
@@ -18,5 +18,5 @@
 {
     DB2, H2, MemSQL, Sybase, SybaseIQ, Composite, Postgres, SqlServer, Hive,
     Snowflake, Presto,
-    BigQuery
+    BigQuery, Spanner
```

Comment:
Not pick - can we please format this list in a consistent way?

Reply:
Sure. You want me to remove the extra space ? Or move BigQuery, Spanner onto the same line as DB2 ?
I kept Spanner, BigQuery on the same line but separate from the rest just to call out the fact they are database supported on the same platform (GCP).

Reply:
actually I don't mind if all are on same line or all on diff line, just looks a bit ugly this way